### DISCUSSION TITLE
MongoDB behind NRPM?

### DISCUSSION THREAD

**Discussion by daVinci13** [NginxProxyManager/nginx-proxy-manager#3727] [General]
Does anyone have a clue on how to set it up? On docker, i run numerous containers, and using nrpm is a delight, however, i want to have a mongodb on a subdomain and i do the usual process, shows host online, but i cant connect to the db! MariaDB works, PostgresDB works, but mongo refuses. Container logs of mongo say that its up and running, but i just cant connect to it.

**Comment by tp1050** [comment]
can you share your compose set up or the npm set up

**Reply by daVinci13** [reply]
I can't, moved away from NRPM.
